build.gradle

```diff
@@ -7,6 +7,7 @@ plugins {
   id 'org.springframework.boot' version '2.2.5.RELEASE'
   id 'org.owasp.dependencycheck' version '5.3.0'
   id 'com.github.ben-manes.versions' version '0.28.0'
+  id "info.solidsoft.pitest" version '1.3.0'
```

Comment:
Could this be upgraded to 1.5.0 and same for the relevent plugins / gradlepitest

Reply:
Ok, can do it.

Reply:
1.5.0 is giving errors that it cannot be found. I checked on the website and the latest version is 1.4.7 so I will configure that one.

Reply:
1.5.0 is not available. The latest version is 1.4.7 but it is complaining about the gradle version. I have taken these settings from a project that was recommended to me by the Platform engineering guy. I will use this for now as it is working fine.